### ISSUE TITLE
v1.3.0: `security: unable to open... for reading: No such file or directory`

### ISSUE THREAD

**Issue by viktorbenei** [bitrise-io/steps-xcode-archive#14]
```
** ARCHIVE SUCCEEDED **
set +v
(i) export_options_path: /Users/vagrant/deploy/export_options.plist
(i) archive_path: /Users/vagrant/deploy/Shoproll Release.xcarchive
==> Collect infos from mobileprovision
security: unable to open "/Users/vagrant/deploy/Shoproll" for reading: No such file or directory
/var/folders/4f/945ywsh506g31f58fpcbg0840000gn/T/bitrise908374675/step_src/generate_export_options.rb:23:in `collect_provision_info': undefined method `each' for nil:NilClass (NoMethodError)
from /var/folders/4f/945ywsh506g31f58fpcbg0840000gn/T/bitrise908374675/step_src/generate_export_options.rb:85:in `'
```

**Comment by viktorbenei** [comment]
Most likely a space / whitespace issue, note the Archive's name: `Shoproll Release.xcarchive` while the "no such file or dir" is compaining about `Shoproll`!

**Comment by viktorbenei** [comment]
Fixed in v1.3.1 https://github.com/bitrise-io/steps-xcode-archive/releases/tag/1.3.1
